android/build.gradle

```diff
@@ -3,17 +3,16 @@ version '1.0-SNAPSHOT'
 
 buildscript {
     ext.detekt_version = '1.14.1'
-    ext.kotlin_version = '1.4.21'
+    ext.kotlin_version = '1.5.0-RC'
```

Comment:
Assuming "RC" is for "release candidate", is it okay to specify a non-stable version here? (considering only the numbered non-suffixed releases stable)

Reply:
I got a warning that I need to move to 1.5.0 but I think I will move back to 1.4.21 indeed